sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -44,7 +50,11 @@ import org.apache.spark.sql.catalyst.plans.logical.{LogicalPlan, Project}
  *                 unless they are specified with full qualified table name with database prefix.
  */
 case class CreateViewCommand(
-    tableDesc: CatalogTable,
+    name: TableIdentifier,
+    userSpecifiedColumns: Seq[(String, Option[String])],
+    comment: Option[String],
+    properties: Map[String, String],
+    originalText: Option[String],
     child: LogicalPlan,
```

Comment:
If we do an explain on this command, the physical plan of this `CREATE VIEW` will return a logicla plan of `child`. Even worse, it could return a parsed plan that has not been analyzed. Do you think we should add a logical plan node for `CreateViewCommand`?